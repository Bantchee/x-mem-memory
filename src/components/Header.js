const Header = (props) => {
    return (
        <div className="row-span-1 flex justify-center content-center bg-yellow-400 relative">
            <h1 className="text-4xl font-bold mt-auto mb-auto">
                X-Men Memory
            </h1>
        </div>
    )
};

export default Header;