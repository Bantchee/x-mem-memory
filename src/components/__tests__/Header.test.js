import React from 'react';
import Header from "../Header";
import { render } from "@testing-library/react"

describe("Header Component", () => {
    test('renders website header', () => {
        const { container } = render(<Header />);
        expect(container).toMatchSnapshot();
    });
});