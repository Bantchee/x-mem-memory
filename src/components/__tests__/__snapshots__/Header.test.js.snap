// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders website header 1`] = `
<div>
  <div
    class="row-span-2 flex justify-center content-center bg-yellow-400 text-neutral-900 relative"
  >
    <h1
      class="text-4xl font-bold mt-auto mb-auto"
    >
      X-Men Memory
    </h1>
  </div>
</div>
`;
